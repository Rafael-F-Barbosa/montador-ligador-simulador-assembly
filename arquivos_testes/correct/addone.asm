 section data
one:        CONST 1
two:        CONST -2


            section text
start:      add     one                 ; Simple add something
            jmp     end
; Let's finish this empty file
end:        stop
     