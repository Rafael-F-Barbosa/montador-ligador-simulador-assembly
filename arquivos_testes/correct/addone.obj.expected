01 05 05 04 14 1 -2